.teamcity/hosted/utils/Utils.kt

```diff
@@ -0,0 +1,133 @@
+package hosted.utils
+
+import hosted.utils.models.ImageInfo
+
+/**
+ * Utilities for the build up of Docker images.
+ */
+class Utils {
+    companion object {
+        /**
+         * Returns .dockerignore context based on given image.
+         * @param info information about target Docker image
+         */
+        fun getDockerignoreCtx(info: ImageInfo): String {
```

Comment:
getDocker**I**gnoreCtx?

Reply:
@burnasheva  `ctx` is a popular abbreviation for "Context" (for example, it's used in multiple Unix kernels), thus I thought it may be appropriate to use it here. Docstrings also clarify the purpose of the method.